src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -30,6 +30,7 @@ case class ClusterRecord(id: Long,
                          operationName: String,
                          status: String,
                          hostIp: Option[String],
+                         creator: String,
```

Comment:
👍  on this column

Reply:
Damn it this is the 22nd column! I want to add another one (to distinguish between the types of service accounts) and can't since `ClusterRecord.tupled` doesn't exist for >22 parameters. 😭

Reply:
Looks like there are workarounds..

Reply:
@blrubenstein pointed out this column would be useful as a label too. Doesn't have to be in this PR.

Reply:
might be time to make the change and break out `NotebookCluster` from `DataprocCluster`

Reply:
Still WIP, but I was able to break apart this case class (thus getting around the slick issue) while keeping the same DB schema.

Reply:
yeah i meant the above as a case class only change